src/styles/blocks/page.scss

```diff
@@ -0,0 +1,14 @@
+.page {
+  height: 100%;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.body {
+  margin: 0;
+  height: 100%;
```

Comment:
Setting `height: 100%` on `.body` may not work as expected if the parent element's height is not defined. Make sure the parent's height is set to 100% or consider using vh units.